metadata.yml

```diff
@@ -1,7 +1,7 @@
 rules:
 - apiGroups: [""]
   apiVersions: ["v1"]
-  resources: ["pods"]
+  resources: ["deployment","replicaset","statefulset","daemonset","replicationcontroller","job","cronjob","pod"]
```

Comment:
the docs inside of the `metadata.yml` have not been updated

Reply:
Yeap...https://github.com/kubewarden/user-group-psp-policy/pull/34#issuecomment-1263494156